README.md

```diff
@@ -20,6 +20,15 @@ configure_args = "-Dusethreads"
 
 ```
 
+Alternatively, you can define 'app' as 'app_github' instead, so the destination 'app' directory will be a git repo clone instead of a directory copy.
+'app_dir_name' is the name of the cloned repo directory under '<output>/app'.
+
+```
+app_github = "https://github.com/gh_user/CoffeeChair.git"
+app_dir_name = "CoffeeChair"
```

Comment:
Wouldn't this always be what the clone will generate? Wouldn't that always be the last part (`File::Basename::basename($github_url) =~ s{\.git$}{}r`?